### ISSUE TITLE
0.9.5 release

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#3746]
Purpose: The release is needed for CM students using Java.
Date: KW14, ideally Thursday, 8th of April

@mpranj do you have time to do a release this week? Otherwise I can also do it once, so that I can also check how the automated releases are like. You can unassign yourself then. I might need your support, though.

### Updates

- [ ] Write release notes
- [ ] rework doc/AUTHORS.md
- [ ] check if website renders correctly
- [ ] Java (Sonartype) Publishing release pipeline

### Increment Version Numbers

- [ ] CMakeLists.txt
- [ ] Change VERSION variable in build-server

**Comment by mpranj** [comment]
I can do a release this Thursday and postpone the open issues to the next milestone.

Is everything relevant to "CM students using Java" already in master or are there any open issues/PRs that need to be finished before release?

**Comment by markus2330** [comment]
Thank you so much!

#3714 is still to be merged and not yet finished. I hope we get something today.

#3747 and #3669 is also relevant for CM students (deadline for writing a plugin is 12.5). Probably we only need docu/CI fixes for these issues (how to install Java on MacOSX/Linux so that libjvm is found).

**Comment by tucek** [comment]
The following issues might be of interest for being addressed before release of 0.9.5: #3759, #3758, #3757, #3710

#3754 is more time consuming than expected, therefore i propose to not depend on it for this release. I will focus on the javadoc improvement issue once the 0.9.5 release is done. We always can do another minor release afterwards.

**Comment by markus2330** [comment]
I suggested to do the release today, because I am only available till today. But as @mpranj is doing the release now, this shouldn't be an issue anymore. @mpranj do you have also time on the weekend? @tucek proposed that he can work until Saturday.

**Comment by mpranj** [comment]
It makes sense to do the release once #3714 is finished, as this is time critical for the CM students.

I can do the release on the weekend. :+1:

**Comment by markus2330** [comment]
Yes, their deadline is on 14.4. Thank you so much :revolving_hearts:

**Comment by tucek** [comment]
See #3714 for unsolved problems regarding a segfault triggered by Java binding test GOptsTest when using default resolver `resolver_mf_xp_x`.

If we still want to release today, we would have to make a decision to either postpone merge of #3714 or disable the mentioned test for the affected environment and open an new issue. (which would be my recommended solution)

**Comment by tucek** [comment]
@mpranj looks like we either need input from @markus2330?

**Comment by mpranj** [comment]
I still plan to do the release today and I don't need any input.

I only waited for #3714 to be finished, as it is a must-have for this release.

**Comment by tucek** [comment]
Should i dissable the test? I am not of any help to debug the problem within the native library in the given timeframe.

**Comment by mpranj** [comment]
> Should i dissable the test? 

No, I forked your changes and will try to debug and fix it. If I am unable to fix it today I can disable the test myself.

Can I assume that #3714 is otherwise finished or are there any further changes?

**Comment by tucek** [comment]
Yes, apart from the segfault issue, #3713 is finished.

**Comment by mpranj** [comment]
Thanks so much to everyone! :heart: 

The release is done, announcements will be published when the website is regenerated with the news.

**Comment by markus2330** [comment]
Thank you so much for getting this done! :rocket: :sparkling_heart:
